src/app/dashboard/page.tsx

```
@@ -1,78 +1,83 @@
 const StatusPanel = dynamic(
-  () => import('@/components/dashboard/StatusPanel').then(mod => ({
-    default: memo(mod.default)
-  })),
+  () =>
+    import('@/components/dashboard/StatusPanel').then(mod => ({
+      default: memo(mod.default),
+    })),
   {
     ssr: false,
-    loading: () => <div className="h-full w-full bg-card animate-pulse rounded-xl"></div>
+    loading: () => <div className="h-full w-full bg-card animate-pulse rounded-xl"></div>,
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)